src/program.js

```diff
@@ -0,0 +1,146 @@
+const { program } = require("commander");
+const package = require("../package.json");
+
+const dicomwebDefaultDir = "~/dicomweb";
+
+program.version(package.version);
+
+/**
+ * Configure commander program. Ideally it should be called just once.
+ * @param {*} configuration Configuration object from command level
+ * @returns Program object
+ */
+function configureProgram(configuration) {
+  const { helpDescription, helpShort, optionsRequired = [] } = configuration;
+
+  program
+    .name(helpShort)
+    .configureHelp({ sortOptions: true })
+    .addHelpText("beforeAll", helpDescription);
+
+  // program command options
+  const options = [
```

Comment:
This list is global and static, shared between all existing command. In the future, each command can create its own subset.